### ISSUE TITLE
Appearing to get false positive error messages for `fqcn[action-core]` with `ansible.builtin.yum`

### ISSUE THREAD

**Issue by JohnVillalovos** [ansible/ansible-lint#4211]
Running ansible-lint 24.6.0

In the log:

```
roles/centos_release_scl_rh/tasks/main.yml:12: fqcn[action-core]: Use FQCN for builtin module actions (ansible.builtin.yum).
```

But the file has:

```
   12   - name: Remove `centos-release-scl-rh`
   13     ansible.builtin.yum:
   14       name: "{{ centos_release_scl_rh_package_name }}"
   15       state: removed
```

Previously this has been working fine.

**Comment by JohnVillalovos** [comment]
Okay. So `ansible.builtin.yum` no longer exists. On IRC I was told that should switch to `ansible.builtin.dnf` even if the OS actually uses `yum`

**Comment by JohnVillalovos** [comment]
FYI:

The `ansible.builtin.yum` module is gone as of Ansible 2.17

https://github.com/ansible/ansible/pull/81895

**Comment by gardar** [comment]
> Okay. So `ansible.builtin.yum` no longer exists. ~On IRC I was told that should switch to `ansible.builtin.dnf` even if the OS actually uses `yum`~
> 
> FYI: Switching to `ansible.builtin.dnf` did not work when connecting to a CentOS 7 system.

I recommend installing `nextgen-yum4` to get `dnf` on EL7, it even performs better than yum ever did.
